uniffi_macros/src/record.rs

```diff
@@ -55,6 +55,10 @@ pub(crate) fn record_ffi_converter_impl(
 ) -> TokenStream {
     let impl_spec = tagged_impl_header("FfiConverter", ident, tag);
     let name = ident_to_string(ident);
+    let mod_path = match mod_path() {
+        Ok(p) => p,
+        Err(e) => return e.into_compile_error(),
+    };
```

Comment:
Maybe make this function return `syn::Result<TokenStream>` instead and handle `into_compile_error` one level higher? Then it should just be a matter of `.unwrap_or_else(syn::Error::into_compile_error)`.